pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
Please revert the changes you've made to this file.

Reply:
<img width="571" alt="image" src="https://user-images.githubusercontent.com/48197694/218037449-3d479e45-63d5-4574-b867-25e7798c34c2.png">

reverted